### ISSUE TITLE
AVT 3 - React Pagination, VO issues with arrow keys and announced text

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3317]
## Environment
macOS Mojave version 10.14.5
Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
Voice Over

## Detailed Description

1. Start VO and use Control-Option - Right Arrow to read across the pagination. 
2. The following is read: "Page number of 6 pages. You are currently on a text element." right before the page number and drop down have focus. This should not be announced.
3. Control-Option - Right Arrow to the page number with drop down and then the following is read, "1, Page number of 6 pages..."  a second time. (see screen shot)
4. When "Page number of 6 pages. You are currently on a text element."  is announced it does not include the page number and it is redundant with the text in the dropdown. 
5. Navigate to the Right and Left arrow keys. 
6. Click the space bar and the page number changes, but it is not announced by VO. 
<img width="1009" alt="Screen Shot 2019-07-09 at 2 24 35 PM" src="https://user-images.githubusercontent.com/21676914/60918001-ce639d00-a257-11e9-8a28-49337eec3a76.png">

**Comment by snidersd** [comment]
I retested this morning and the issue is no longer present. It looks like it may have been fixed in #4756
